### PR TITLE
Add workflow to label automatically the PR based on the changed package

### PR DESCRIPTION
**Pull request by lucacavallaro**
#### List of Changes

1. Add a new `GitHub Workflow`, `.github/workflows/label.yml` (https://github.com/actions/labeler)

### CODE REVIEW

**Review by gunzip — APPROVED** [review]

File: .github/labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+io-func-sign-issuer: apps/io-func-sign-issuer/**/*
```
Comment (gunzip) [inline .github/labeler.yaml:1]: it looks like the action is looking for labeler.yml (not labeler.yaml)

Comment (lucacavallaro) [reply .github/labeler.yaml:1]: great! where you found out?

Comment (gunzip) [reply .github/labeler.yaml:1]: in the action output

Comment (lucacavallaro) [reply .github/labeler.yaml]: 🤦 🤦 🤦

**Pull request merged**